src/Classes/AbstractOrFinalRule.php

```diff
@@ -19,6 +19,21 @@
 
 final class AbstractOrFinalRule implements Rule
 {
+    /**
+     * @var string[]
+     */
+    private $excludedClassNames;
+
+    /**
+     * @param string[] $excludedClassNames
+     */
+    public function __construct(array $excludedClassNames = [])
+    {
+        $this->excludedClassNames = \array_map(function (string $excludedClassName): string {
```

Comment:
You can declare this closure static:
```php
static function (string $excludedClassName): string {
```

Reply:
Fixing with https://github.com/localheinz/php-cs-fixer-config/pull/157.